backend/amd64/emit.mlp

```diff
@@ -2219,7 +2277,10 @@ let emit_trap_notes () =
     | S_macosx -> false  (* can be supported with a symbol *)
     | S_gnu | S_solaris | S_linux_elf | S_bsd_elf
     | S_beos | S_linux -> true
-    | S_cygwin | S_mingw | S_mingw64 | S_win64 | S_win32 | S_unknown ->
+    | S_cygwin | S_mingw | S_mingw64 | S_win64 | S_win32 | S_unknown
+    | S_freebsd| S_netbsd | S_openbsd ->
+      (* CR xclerc for xclerc: double check whether {Free,Net,Open}BSD
```

Comment:
They are all ELF now I believe so it would work but we don't need to support them here and have no way to test. Maybe just replace the CR with this comment.